feat: photos with gatsby-plugin-image

jurek2006 opened the pull request.

jurek2006 committed:
feat: photos with gatsby-plugin-image

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report

:triangular_flag_on_post: Your build failed. [See the build logs here](https://gatsbyjs.com/dashboard/2bed36c5-2876-418a-9a18-97f3b59e23f1/sites/37b4728e-2060-454b-9615-bac3fb991609/builds/9352b6ae-fee6-43ab-85e2-7925f0934c2b/details)

**Errors**
<pre>There was an error in your GraphQL query:

Unknown argument "slug" on field "formiumForm" of type "Query".

GraphQL request:3:17
2 |   {
3 |     formiumForm(slug: { eq: "contactus" }) {
  |                 ^
4 |       id</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "createAt" on type "FormiumForm".

If you don't expect "createAt" to exist on the type "FormiumForm" it is most likely a typo.
However, if you expect "createAt" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "createAt" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "FormiumForm":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "name" on type "FormiumForm".

If you don't expect "name" to exist on the type "FormiumForm" it is most likely a typo.
However, if you expect "name" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "name" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "FormiumForm":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "slug" on type "FormiumForm".

If you don't expect "slug" to exist on the type "FormiumForm" it is most likely a typo.
However, if you expect "slug" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "slug" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "FormiumForm":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "projectId" on type "FormiumForm".

If you don't expect "projectId" to exist on the type "FormiumForm" it is most likely a typo.
However, if you expect "projectId" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "projectId" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "FormiumForm":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "updateAt" on type "FormiumForm".

If you don't expect "updateAt" to exist on the type "FormiumForm" it is most likely a typo.
However, if you expect "updateAt" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "updateAt" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "FormiumForm":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "version" on type "FormiumForm".

If you don't expect "version" to exist on the type "FormiumForm" it is most likely a typo.
However, if you expect "version" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "version" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "FormiumForm":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions</pre>

jurek2006 committed:
feat: gatsby-plugin-image

Added image on index with gatsby-plugin-image (static) but for images from contentful used old gatsby-image

jurek2006 committed:
feat: images from contentful with gatsby-image

Added images from contentful used old gatsby-image

jurek2006 committed:
Merge branch 'p08-gatsby-image' of github.com:jurek2006/100days-of-code-gatsby2021_project into p08-gatsby-image

jurek2006 commented:
Error in checking with Gatsby Cloud (with graphQL and Formium)
